<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trakem2_stitching_dialog_ui</class>
 <widget class="QDialog" name="trakem2_stitching_dialog_ui">
  <property name="windowTitle">
   <string>TrakEM2 Settings</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Current Pixel Ratio [pixels/μm]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="qsbCurrentPixelRatio">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip">
          <string>Current pixel ratio</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-999999.989999999990687</double>
         </property>
         <property name="maximum">
          <double>999999.989999999990687</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="displacement_vectors_selector" name="displacement_vectors" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string> Step Size [μm]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="qsb_step_size">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTrakem2Calibration">
         <property name="toolTip">
          <string>Perform calibration and output alignment file</string>
         </property>
         <property name="text">
          <string>Calibrate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnWriteTrakem">
         <property name="toolTip">
          <string>Write TrakEM2 for loaded settings</string>
         </property>
         <property name="text">
          <string>Write TrakEM2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>displacement_vectors_selector</class>
   <extends>QWidget</extends>
   <header>displacement_vectors_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="slim5.qrc"/>
 </resources>
 <connections/>
</ui>
